orm/encoding/ormfield/codec_test.go

```diff
@@ -85,3 +85,55 @@ func TestNTBytesTooLong(t *testing.T) {
 	_, err = cdc.ComputeBufferSize(bz)
 	assert.ErrorContains(t, err, ormerrors.BytesFieldTooLong.Error())
 }
+
+func TestCompactUInt32(t *testing.T) {
+	rapid.Check(t, func(t *rapid.T) {
```

Comment:
These rapid tests are useful, but I think this is an example where using hardcoded value might help. Could you add 2-3 examples of hardcoded values (e.g. around the 2-bit checkpoints) and their expected bytes array?

Reply:
added some manual test cases